src/main/java/com/cairn/waypoint/dashboard/endpoints/protocol/DeleteProtocolEndpoint.java

```diff
@@ -81,10 +81,10 @@ public ResponseEntity<String> generateSuccessResponse(
         note.setActive(Boolean.FALSE);
         note.setModifiedBy(modifiedBy);
       });
-      step.getLinkedHomework().forEach(homeworkLink -> {
+        /*step.getLinkedHomework().forEach(homeworkLink -> {
```

Comment:
Will forego adding the commented code removal comment but let's do this globally. You could do a global search in the project for `//` or `/*` to find and remove all your comments within the codebase.